.gitignore

```diff
@@ -1,4 +1,5 @@
 # Godot 4+ specific ignores
 .godot/
 .vscode/
-*.tmp
\ No newline at end of file
+*.tmp
+*[Tt]est*
```

Comment:
Can we change this to have `.tscn` at the end? Since we only want to ignore scenes as far as I understand

Reply:
i will, ill do all the changes and then ill let you know when im done